__tests__/github-mention-test.js

```diff
@@ -54,6 +54,8 @@ describe('Github Mention', function() {
         3238,
         'mention-bot',
         'master',
+        false,
```

Comment:
Can you add an inline comment to say what this argument is.

false, // whatever this is